### PR TITLE
Refactor Go Proxies

### PR DESCRIPTION
**Pull request by ZachGarcia42**

### CODE REVIEW

**Comment by ZachGarcia42** [comment]
* **#137** <a href="https://app.graphite.dev/github/pr/cornellrocketryteam/Ground-Software/137?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @ZachGarcia42 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: go-proxies/fill-telem-proxy/main.go [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+	"time"
+
+	"github.com/cornellrocketryteam/Ground-Software/go-proxies/pkg/types"
+
+	pb "github.com/cornellrocketryteam/Ground-Software/go-proxies/proto-out"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+var datastore types.Datastore
```
Comment (maxslarsson) [inline go-proxies/fill-telem-proxy/main.go:18]: Does this have to be global? Or is it better for it to be global?

File: go-proxies/websocket-proxy/main.go [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+func main() {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	// Initialize datastore
+	datastore.Init(ctx)
+
+	// Start the broadcaster Goroutine
+	go func() {
+		ticker := time.NewTicker(time.Second)
```
Comment (maxslarsson) [inline go-proxies/websocket-proxy/main.go:131]: Could we try doing every 0.5 seconds instead or even slightly faster? I think the 750ms we had for last version of ground station was kind of slow. Like it didn't feel realtime

**Review by maxslarsson — CHANGES_REQUESTED** [review]

File: go-proxies/websocket-proxy/main.go (hunk 2 above)
Comment (ZachGarcia42) [reply go-proxies/websocket-proxy/main.go:131]: Yeah, we can adjust that as much as we like

File: go-proxies/fill-telem-proxy/main.go (hunk 1 above)
Comment (ZachGarcia42) [reply go-proxies/fill-telem-proxy/main.go]: Good catch. Moved to inside main

File: go-proxies/websocket-proxy/main.go (hunk 2 above)
Comment (ZachGarcia42) [reply go-proxies/websocket-proxy/main.go]: Updated to every 0.5 seconds. Last version was every 1 second I believe

**Comment by ZachGarcia42** [comment]
### Merge activity

* **Nov 7, 1:59 PM EST**: A user started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/cornellrocketryteam/Ground-Software/137).

**Pull request merged**